### ISSUE TITLE
Get-IntuneWin32AppAssignment -GroupName returns "Cannot validate argument on parameter 'Exception'

### ISSUE THREAD

**Issue by ferlessleedr** [MSEndpointMgr/IntuneWin32App#108]
The following is the full text return of running this command in the console of an elevated PowerShell ISE instance:

`PS C:\Windows\system32> Get-IntuneWin32AppAssignment -GroupName "AllApps"

WARNING: An error occurred while resolving groupId for assignment with ID: 93b73e3f-52a0-4d98-b602-a51146a7ce24_0_0. Error message: Cannot validate argument on parameter 'Exception'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.

WARNING: An error occurred while resolving groupId for assignment with ID: 6a5d8740-dad7-4460-b842-1d8c024e65ac_0_0. Error message: Cannot validate argument on parameter 'Exception'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.

WARNING: An error occurred while resolving groupId for assignment with ID: 93b73e3f-52a0-4d98-b602-a51146a7ce24_0_0. Error message: Cannot validate argument on parameter 'Exception'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.

WARNING: An error occurred while resolving groupId for assignment with ID: 3932b7b1-e81e-4e70-85ec-64f7bd74643a_0_0. Error message: Cannot validate argument on parameter 'Exception'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.

WARNING: An error occurred while resolving groupId for assignment with ID: 93b73e3f-52a0-4d98-b602-a51146a7ce24_0_0. Error message: Cannot validate argument on parameter 'Exception'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.

WARNING: An error occurred while resolving groupId for assignment with ID: 6a5d8740-dad7-4460-b842-1d8c024e65ac_0_0. Error message: Cannot validate argument on parameter 'Exception'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.

WARNING: An error occurred while resolving groupId for assignment with ID: 93b73e3f-52a0-4d98-b602-a51146a7ce24_0_0. Error message: Cannot validate argument on parameter 'Exception'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.

WARNING: An error occurred while resolving groupId for assignment with ID: 6a5d8740-dad7-4460-b842-1d8c024e65ac_0_0. Error message: Cannot validate argument on parameter 'Exception'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.

WARNING: An error occurred while resolving groupId for assignment with ID: 93b73e3f-52a0-4d98-b602-a51146a7ce24_0_0. Error message: Cannot validate argument on parameter 'Exception'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.

WARNING: An error occurred while resolving groupId for assignment with ID: 01610b33-7869-4290-bf2a-07559d4ae3d4_1_0. Error message: Cannot validate argument on parameter 'Exception'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
`

It then keeps going like this for quite a while, repeating group IDs until something triggers it to stop.  Different groups will return different numbers of lines but I've tried this with numerous groups I know I'm using including "All Users" and "All Devices".  I've set the command up via the Commands sidebar in the ISE and then inserted and run it from there and it's happening the same way.  Is there something I'm doing wrong here, or is there an issue with the command?  It seems incredibly useful, to be able to make sure that I'm not using a group anywhere in Intune before deleting it for example.

**Comment by NickolajA** [comment]
Yeah this function clearly needs some updating. I'll address this in 1.4.2. Thank you for reporting!
